README.md

```diff
@@ -58,6 +58,8 @@ It was designed with the following vision in mind:
 pip install pybit
 ```
 
+If you want to use only v5 ByBit APIs, you can use [v5 branch](https://github.com/bybit-exchange/pybit/tree/v5-only)
```

Comment:
```suggestion
If you want to use only v5 ByBit APIs (unofficial fork version), you can use [v5 branch](pip install git+https://github.com/ervuks/pybit.git@v5-only)
```

Reply:
This is not supposed to move people to my fork. There already is a v5 only branch in repo by bybit-exchange